pandas_gbq/exceptions.py

```diff
@@ -3,6 +3,14 @@
 # license that can be found in the LICENSE file.
 
 
+class GenericGBQException(ValueError):
+    """
+    Raised when an unrecognized Google API Error occurs.
+    """
+
+    pass
```

Comment:
(nit)
`pass` is not needed here, can be removed (just like in `ConversionError`).

Reply:
Done